src/components/Composer/Editor/index.tsx

```diff
@@ -108,25 +77,29 @@ export default function Editor() {
         <ToolbarPlugin />
         <RichTextPlugin
           contentEditable={
-            <ContentEditable
-              className="
-          block w-[97%] m-4 box-border h-20 z-10 overflow-auto"
-            />
+            <ContentEditable className="block text-lg mt-4 box-border h-20 z-10 overflow-auto" />
           }
           placeholder={
-            <div className="absolute z-0 text-gray-400 pointer-events-none select-text top-12 left-4 right-3 whitespace-nowrap">
+            <div className="absolute z-0 text-gray-400 text-lg pointer-events-none top-12 whitespace-nowrap">
               What's happening?
             </div>
           }
-          ErrorBoundary={ErrorBoundary}
+          ErrorBoundary={() => <div>{ERROR_MESSAGE}</div>}
+        />
+        <OnChangePlugin
+          onChange={(editorState) => {
+            editorState.read(() => {
+              const markdown = $convertToMarkdownString(TRANSFORMERS);
+              setPublicationContent(markdown);
+            });
+          }}
         />
-        <OnChangePlugin onChange={onChange} />
         <HistoryPlugin />
-        <ListPlugin />
```

Comment:
This plugin allows you to nest the list and break out of list in markdown? Did you test it?

Reply:
Yes there is no limit of nesting :( sed

Reply:
fixed